bucket/python.json

```diff
@@ -3,14 +3,31 @@
     "description": "A programming language that lets you work quickly and integrate systems more effectively.",
     "license": "Python-2.0",
     "version": "3.8.2",
+    "notes": "Allow applications and third-party installers to find python by running: $dir\\install-pep-514.reg",
     "architecture": {
         "64bit": {
-            "url": "https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe",
-            "hash": "8e400e3f32cdcb746e62e0db4d3ae4cba1f927141ebc4d0d5a4006b0daee8921"
+            "url": [
+                "https://raw.githubusercontent.com/ScoopInstaller/Main/master/scripts/python/install-pep-514.reg",
+                "https://raw.githubusercontent.com/ScoopInstaller/Main/master/scripts/python/uninstall-pep-514.reg",
+                "https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe"
```

Comment:
First item in array

Reply:
[`installer` requires](https://github.com/lukesampson/scoop/wiki/App-Manifests) either the installer is the last URL or to explicitly specify the name:

> file: The installer executable file. For installer this defaults to the last URL downloaded. Must be specified for uninstaller.


The installer name is different for each architecture and each version. Instead of trying to write confusing logic to make that work, I made it the last one. Since that's a general scoop app restriction, I assumed this would be a common convention?